content/physical_properties/electrical_conductivity/electrical_conductivity_lab_setup_measurements.rst

```diff
@@ -1,225 +1,209 @@
-of the lab set-up is from 0.016 Hz - 1 MHz (logarithmically-sampled).
+Electrical conductivity/resistivity measurements are an integral part of classifying physical rock properties. Here, we present the general approach for measuring a rock’s electrical properties, including electrical conductivity/resistivity and chargeability. Instrumentation is also discussed.
 
-.. figure:: ./images/GSCcompleximpedance.png
-   :figwidth: 50%
+Two-Electrode Laboratory Setup
+------------------------------
+
+Electrical resistivity measurements are most commonly performed using a two-electrode setup. This is illustrated in :numref:`fig_cond_lab_setup`. For these measurements, a rock core of known dimensions is placed between two copper or graphite electrodes. Current is then forced through the rock, which acts as the resistive element for an electrical circuit. By measuring the voltage drop across the rock, Ohm's law can be used to determine the corresponding resistance. Given that we know the dimensions of the rock sample, Pouillet's law can be used to obtain the rock's electrical resistivity. The electrical conductivity is obtained by taking the reciprocal of the resistivity.
+
+Electrical resistivity measurements can be performed in both the frequency domain and the time domain. For frequency domain measurements, a sinuisoidal current is driven through the rock sample. In the time domain, a boxcar waveform is used.
+
+.. figure:: ./images/cond_lab_setup.png
+   :figwidth: 75%
    :align: center
-   :name: GSCcompleximpedance
+   :name: fig_cond_lab_setup
+
+   Basic setup for electrical resistivity measurements. (A) Theoretical circuit where rock acts as a a resistive element. (B) Rock sample between two electrodes. (C) Circuit schematic for frequency domain and time domain measurements.
 
-   Complex impedance measurement system at Geological Survey of Canada (GSC).
+DC Conductivity/Resistivity
+---------------------------
 
-.. figure:: ./images/ACcircuit_simple.png
-   :figwidth: 30%
-   :align: left
-   :name: ACcircuit_simple
+.. figure:: ./images/DCcircuit_simple.png
+   :figwidth: 25%
+   :align: right
+   :name: fig_DC_measurements
 
-   A simple AC circuit.
+   Electrical circuit for DC resistivity measurements.
 
-An equivalent circuit diagram for the lab measurements is shown in
-:numref:`ACcircuit_simple`. For a source that is providing a voltage, the
-currents depend upon the conductivity of the rock. The electrical impedance is
-defined as
+The DC (or zero-frequency) resistivity is obtained by forcing direct current (:math:`I`) through the rock and measuring the voltage drop (:math:`V`). Using Ohm's law, the resistance (:math:`R`) due to the rock is given by:
 
 .. math::
+   R = \frac{V}{I}
 
-   Z(\omega) = \frac{V(\omega)}{I(\omega)}
+According to Pouillet's law, the electrical resistivity of the sample is the product of the resistance and its cross-sectional area (:math:`A`), divided by the sample's length (:math:`l`):
```

Comment:
again, I have not seen this commonly named in literature, I would suggest starting the sentence with `The electrical resistivity of the sample ...` and if you would like to include the name, perhaps use a parenthetical 
```
The electrical resistivity of the sample is the product of the resistance and its cross-sectional area (:math:`A`), divided by the sample's length (:math:`l`)

.. math::
    \rho = \frac{RA}{l}

(sometimes also referred to as Pouillet's law) 
```